src/Validation/RequestValidator.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+
+namespace WMDE\Fundraising\Frontend\Validation;
+
+use WMDE\Fundraising\Entities\Request;
```

Comment:
The name of this entity is really unfortunate. Request already has a meaning in web apps, so now things such as this RequestValidator require a double take.